philistine/mne/io.py

```diff
@@ -43,42 +43,44 @@ def write_raw_brainvision(raw, vhdr_fname, events=True):
 
     Notes
     -----
-    The BrainVision format supports by-channel filter and measurement information and moreover distinguishes between hardware and software filters. MNE does neither.
-    Currently, filter information is not exported.
-
-    Moreover BrainVision also allows for more complex trigger codes than MNE's simple
-    integers, e.g. distinguishing on supported hardware between stimulus codes (prefixed by
-    an S) and responses codes (prefixed by an R). MNE's numeric events are all treated as
-    'stimulus markers' and prefixed by an S on output.
-
-    In other words, a round trip import-export is a lossy operation in terms of metadata.
-    The actual EEG recording should be losslessly preserved within the realm of floating
-    point precision.
+    The Brainvision format supports by-channel filter and measurement
+    information and moreover distinguishes between hardware and software
+    filters. MNE does neither. Currently, filter information is not exported.
+
+    Moreover Brainvision also allows for more complex trigger codes than MNE's
+    simple integers, e.g. distinguishing on supported hardware between stimulus
+    codes (prefixed by an S) and responses codes (prefixed by an R). MNE's
+    numeric events are all treated as 'stimulus markers' and prefixed by an S
+    on output.
+
+    In other words, a round trip import-export is a lossy operation in terms of
+    metadata. The actual EEG recording should be losslessly preserved within
+    the realm of floating point precision.
     """
 
     vmrk_fname = vhdr_fname[:-4] + 'vmrk'
     eeg_fname = vhdr_fname[:-4] + 'eeg'
 
     if isinstance(events, np.ndarray):
         pass
-    elif events == False:
-        events = np.ndarray([0,3])
-    elif events == True:
-        events = mne.find_events(raw,verbose=False)
+    elif not events:
+        events = np.ndarray([0, 3])
+    elif events:
+        events = mne.find_events(raw, verbose=False)
     else:
         raise ValueError('events must be boolean or 3 x n_events ndarray.')
 
     # eliminate the stim channel
-    raw = raw.copy().pick_types(eeg=True, eog=True, meg=True, misc=True, stim=False)
+    raw = raw.copy().pick_types(eeg=True, eog=True, meg=True, misc=True,
+                                stim=False)
 
     _write_vmrk_file(vmrk_fname, eeg_fname, events)
     _write_vhdr_file(vhdr_fname, vmrk_fname, eeg_fname, raw)
     _write_bveeg_file(eeg_fname, raw)
 
 
 def _write_vmrk_file(vmrk_fname, eeg_fname, events):
-    """ Write BrainvVision marker file)
-    """
+    """Write Brainvision marker file."""
```

Comment:
The string terminator on the next line was a reminder to myself to add more documentation here ... but I haven't gotten around to it yet and it's a protected member, so let's keep flake8 from complaining.